projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -742,6 +762,8 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit {
     }
 
     ngAfterViewInit(): void {
-        this.updatePagination();
+        setTimeout(() => {
```

Comment:
If this is only because of a value that changed after rendering, the error is just a warning. I think using a setTimeout to get rid of a warning is replacing one smaller problem with a bigger problem.

We should try to reproduce that problem in our examples app

Reply:
I recreated the error in VCD-UI. It happens when you are on a screen where the magic page size calculates a new page size in the after view init method. However, I tried for ~1 hour to reproduce this error in our examples app. I tried to have a component where the height of the datagrid is based on the height of the surrounding content. I got it so the pageSize was changed in the AfterViewInit method, but the error was never thrown. I could not figure out why this was the case.